LoopBackTests/LBUserTests.m

```diff
@@ -81,7 +81,7 @@ - (void)setUp {
     NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
     [defaults removeObjectForKey:DEFAULTS_CURRENT_USER_ID_KEY];
 
-    self.adapter = [LBRESTAdapter adapterWithURL:[NSURL URLWithString:@"http://localhost:3000"]];
+    self.adapter = [LBRESTAdapter adapterWithURL:[NSURL URLWithString:@"http://localhost:3000/api"]];
```

Comment:
The test server's `restApiRoot` is not set to `/api` but rather just `/`. So, the unit tests won't succeed with this modification.